### ISSUE TITLE
Line break in heredoc is not honoured

### ISSUE THREAD

**Issue by melcloud** [segmentio/terraform-docs#210]
### Prerequisites

Put an `x` into the box that applies:

- [x] This issue describes a bug.
- [ ] This issue describes a feature request.

For more information, see the [Contributing Guidelines](https://github.com/segmentio/terraform-docs/tree/master/CONTRIBUTING.md).

### Description

String in terraform heredoc requires extra line break to render properly in markdown. This was fixed in 0.8.0. But broken again in 0.8.2. See https://github.com/segmentio/terraform-docs/issues/167

#### Actual Behavior

Given follow variable, I expect the description to be formatted as shown in markdown. However, no line break is rendered between image: and uid:

pretty version works as expected

```
variable "envoy_config" {
  description = <<-EOT
    Configuration for Envoy proxy container

    image:            The image tag in ECR for appmesh envoy. Default to aws-appmesh-envoy:v1.12.1.1-prod
    uid:              The user identifier used to run container under a specific user. Default to 1337
    ingress_port:     The port used by envoy to accept inbound traffic. Any requests to the contianer port will be redirected to this
    egress_port:      The port used by envoy for outbound traffic. Any outboud traffic will be redirected to this
    log_level:        The log level of envy. Default to info
    egress_ignore_ip: The egress traffic going to the specified IP addresses is ignored and not redirected to envoy_egress_port
  EOT
  type        = object({ image = string, uid = string, ingress_port = number, egress_port = number, log_level = string, egress_ignore_ip = list(string) })
  default = {
    image            = "aws-appmesh-envoy:v1.12.1.1-prod",
    uid              = "1337",
    ingress_port     = 15000,
    egress_port      = 15001,
    log_level        = "error",
    egress_ignore_ip = ["169.254.170.2", "169.254.169.254"]
  }
}
```

Do you have long logs to share? Please go to https://ghostbin.com and insert the link here.

#### Expected Behavior

Render new lines as it is in heredoc for markdown

```
```

### Replication Case

Tell us which steps to take to replicate your problem.

1. ...
2. ...
3. ...

### Version

terraform-docs version v0.0.0- v0.8.2 darwin/amd64 BuildDate: 2020-02-03T18:39:25+0000

**Comment by khos2ow** [comment]
Hmm, strange. How did you test this, with downloading the released binary or building the HEAD?
This is supposed to be fixed in #169, specifically I added these lines to example and tests keep passing. https://github.com/segmentio/terraform-docs/blob/5a05b953496081556fd4c723161994c3f83b06d7/examples/main.tf#L9-L10

The way it's supposed to work is if consecutive lines are:

1) of a paragraph: double `<space>` will be added to the end of each lines to make markdown engine treat them as a "carriage return"
2) of a markdown list: no extra `<space>` will be added, because markdown will know what to do with them.

I'm in the middle of adding extensive test cases in #208 and I'll make sure to add specific test case for this in it, and if need be fix it, otherwise I suspect you might have a mechanism (e.g. `.editorconfig`) to strip put extra spaces from `.md` files which will be problematic in this case and I'd suggest to make sure that's not the case.

**Comment by melcloud** [comment]
Hi @khos2ow,

Thanks for looking into this. I use the binary directly downloaded from github Release.

I also think I find out which works, which not.

In the below snapshot, `Key2` works as expected, `Foo1` and `Foo2` are on different lines. 
`Key` doesn't work, as there are two spaces in front of `Foo1` and `Foo2`.

Both worked in version 0.8.0.

```hcl
variable "test" {
  description = <<-EOT
    This is is a multline test which works

    Key
      Foo1: blah
      Foo2: blah

    Key2
    Foo1: bar1
    Foo2: bar2
  EOT
  type = string
  default = ""
}
```

**Comment by khos2ow** [comment]
That's interesting, I added a test and a fix for this particular case (i.e. consecutive lines with whitespace at the beginning of the lines, [pkg/tmpl/sanitizer_test.go#L245-L256](https://github.com/segmentio/terraform-docs/blob/master/pkg/tmpl/sanitizer_test.go#L245-L256) and [pkg/tmpl/testdata/multiline/indentations.golden](https://github.com/segmentio/terraform-docs/blob/master/pkg/tmpl/testdata/multiline/indentations.golden)).

Can you try to build the `HEAD` of master and try to see if you still have the problem?

**Comment by melcloud** [comment]
Yes, it works now. Thank you!

**Comment by khos2ow** [comment]
Fixed as part of #208
